messaging/rust/src/messaging.rs

```diff
@@ -271,6 +271,14 @@ impl MessagingSender<wasmbus_rpc::actor::prelude::WasmHost> {
             wasmbus_rpc::actor::prelude::WasmHost::to_provider("wasmcloud:messaging", link_name)?;
         Ok(Self { transport })
     }
+
+    /// Constructs a client for actor-to-actor messaging
+    /// using the recipient actor's public key
+    pub fn to_actor(actor_id: &str) -> Self {
```

Comment:
Again I'm leery of exposing a function like this because it makes it sound like actor-to-actor communication across the messaging interface is commonplace when the interface really isn't designed to work this way.